### PR TITLE
Swap to WOFF2 variable fonts, add CLS fallbacks

### PR DESCRIPTION
**Pull request by 0aveRyan**
* Reduces `/fonts` by about 50% without removing any 💪
* Switches to web.dev recommended WOFF2 format in variable fonts 🚀
* Adds fontFamily fallbacks to [system font stacks](https://github.com/system-fonts/modern-font-stacks) that most-reduce CLS and maintain design intentions 🦸
* Purposefully didn't adopt variable fonts for Poppins or Merriweather where the file size tradeoff is significant to consolidate single weights to variable and few users would benefit from single-asset usage with one/few weights in use -- would need most weights in-use to benefit.

_(Replacement for #19 blending new theme.json changes)_

### CODE REVIEW

File: theme.json [hunk 1]
```diff
@@ -190,23 +190,23 @@
...
 							"src": [
-								"file:./assets/fonts/Mulish-Italic-VariableFont_wght.ttf"
+								"file:./assets/fonts/Mulish-Italic-VariableFont_wght.woff2"
 							]
 						}
 					],
-					"fontFamily": "Mulish",
-					"name": "Mulish, sans-serif",
+					"fontFamily": "Mulish, Seravek, \"Gill Sans Nova\", Ubuntu, Calibri, \"DejaVu Sans\", source-sans-pro, sans-serif",
+					"name": "Mulishf",
```
Comment (arunshenoy99) [inline theme.json:209]: Did you mean `Mulish`?

File: theme.json [hunk 2]
```diff
@@ -793,110 +711,28 @@
...
-							"fontFamily": "Playfair Display",
-							"fontStyle": "italic",
-							"fontWeight": "800",
-							"src": [
-								"file:./assets/fonts/playfair-display-v29-latin-800italic.woff2"
-							]
-						},
-						{
-							"fontFamily": "Playfair Display",
+							"fontFamily": "Playfair Displa",
```
Comment (arunshenoy99) [inline theme.json:721]: Did you mean `Playfair Display`?

File: theme.json [hunk 3]
```diff
@@ -292,12 +292,12 @@
...
 							"fontWeight": "100 900",
 							"src": [
-								"file:./assets/fonts/Montserrat-Italic-VariableFont_wght.ttf"
+								"file:./assets/fonts/Montserrat-Italic-VariableFont_wght.woff2"
 							]
 						}
 					],
-					"fontFamily": "Montserrat",
-					"name": "Montserrat, sans-serif",
+					"fontFamily": "Montserrat, ui-rounded, \"Hiragino Maru Gothic ProN\", Quicksand, Comfortaa, Manjari, \"Arial Rounded MT\", \"Arial Rounded MT Bold\", Calibri, source-sans-pro, sans-",
```
Comment (arunshenoy99) [inline theme.json:299]: Did you mean `sans-serif` instead of `sans-`?

**Comment by francgrasso** [comment]
I'm going to merge the PR and then add the fixed highlithed by @arunshenoy99

**Comment by 0aveRyan** [comment]
I don't know how I clipped those second pass, good catches @arunshenoy99 😎

**Pull request merged**